### PR TITLE
Use admin-sortable2 for JobOpenings

### PR DESCRIPTION
**Pull request by mkoistinen**
Due to issues with Parler, it is rather complicated to display Job Openings alphabetically in all languages, so, let's let the user declare their order.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
     'unidecode',
     'django-multiupload>=0.3',
     'django-sortedm2m',
+    'django-admin-sortable2>=0.5.0',
```
Comment (czpython) [inline setup.py]: need also to add the app to addon.json no?

Comment (mkoistinen) [inline setup.py]: Yes, thank you.

File: addon.json [hunk 2]
```diff
@@ -8,6 +8,7 @@
         "standard_form",
         "aldryn_common",
         "aldryn_boilerplates",
-        "aldryn_apphooks_config"
+        "aldryn_apphooks_config".
```
Comment (cyberstar) [inline addon.json]: probably comma needed instead of dot.

**Comment by cyberstar** [comment]
LGTM :+1:

**Pull request merged**